.codecov.yml

```diff
@@ -7,4 +7,4 @@ coverage:
         target: 80%
     project:
       default:
-        target: auto
+        target: 80%
```

Comment:
This fixes this

![image](https://user-images.githubusercontent.com/172584/78215350-e536d780-74e9-11ea-8a84-53a246b3de93.png)